src/panels/lovelace/editor/card-editor/hui-card-picker.ts

```diff
@@ -262,6 +253,17 @@ export class HuiCardPicker extends LitElement {
     this._loadCards();
   }
 
+  protected updated(changedProps) {
+    super.updated(changedProps);
+    if (this._prevFilter !== this._filter) {
+      this._prevFilter = this._filter;
```

Comment:
You should be able to this just based on `changedProps`. `changedProps.has('_filter') I think.